Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM node:alpine
+
+LABEL io.kuzzle.vendor="Kuzzle <support@kuzzle.io>"
+LABEL description="Documentation website with running tests"
+
+WORKDIR /var/app
+
+COPY . /var/app
+
+RUN set -eux \
+  \
+  && npm install \
+  && npm i node-sass
```

Comment:
isn't it already installed by the `npm install` above?

Reply:
I don't know why it doesn't get installed if I don't do it twice...